astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+# Originally authored by Jazmin Berlanga Medina
+# (jazmin.berlanga@gmail.com), with input from
+# Adrian Price-Whelan (email), Erik Tollerud (email),
+# Christoph Deil (email), Eric Jeschke (email) and Brett Morris (email).
```

Comment:
Note that we generally do not leave comments of authorship like this: the github changelog is used to keep track of who contributed to what.  That makes it much easier to keep it up-to-date: no one has to change text like this.  To give credit for something like "maintainer" or "main developer" like you would be for this, text belongs better in actual visible documentation where *users* will actually see it.